brand/README.md

```diff
@@ -3,42 +3,9 @@
 
 # Falco Branding Guidelines
 
-This document describes The Falco Project's branding guidelines, language, and message.
+Falco is an open source security project whose brand and identity are governed by the [Cloud Native Computing Foundation](https://www.linuxfoundation.org/legal/trademark-usage).
 
-Content in this document can be used to publicly share about Falco.
-
-
-
-### Logo
-
-There are 3 logos available for use in this directory. Use the primary logo unless required otherwise due to background issues, or printing.
-
-The Falco logo is Apache 2 licensed and free to use in media and publication for the CNCF Falco project.
-
-### Colors
-
-| Name      | PMS  | RGB         |
-|-----------|------|-------------|
-| Teal      | 3125 |   0 174 199 |
-| Cool Gray |   11 |  83  86  90 |
-| Black     |      |   0   0   0 |
-| Blue-Gray | 7700 |  22 92 125  |
-| Gold      | 1375 | 255 158  27 |
-| Orange    |  171 | 255  92  57 |
-| Emerald   | 3278 |   0 155 119 |
-| Green     |  360 | 108 194  74 |
-
-The primary colors are those in the first two rows.
-
-### Slogan
-
-> Cloud Native Runtime Security
-
-### What is Falco?
-
-Falco is a runtime security project originally created by Sysdig, Inc.
-Falco was contributed to the CNCF in October 2018.
-The CNCF now owns The Falco Project.
+Please see the [Falco Branding](https://falco.org/falco-brand) page on our website for full details and guidelines.
```

Comment:
```suggestion
This document describes the official branding guidelines of The Falco Project. Please see the [Falco Branding](https://falco.org/falco-brand) page on our website for further details.
```

IMO, it's ok to add a link that points to the website for further details, but the primary source of information (the official one) must be kept here. So my suggestion is to keep (and eventually update) at least the following sections:
- Logo
- Colors
- Slogan
- What is Falco?

PS
I'm conscious this may create redundancy between this repository and the website, but it's acceptable.